### PR TITLE
Mark patch 12

### PR DESCRIPTION
**Pull request by markodonnell-deriv**
Added deriv-com-v2 tests.
Plus prefixed global command with c_

### CODE REVIEW

File: .DS_Store
Comment (habib-deriv) [inline .DS_Store:1]: Hi @markodonnell-deriv I think, this file should be in .gitignore?

File: cypress/e2e/smoke/home.cy.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ describe('QATEST-1279 - Navigation Responsive - Menu items - EU and ROW (Includi
 
     //Click through menu-items and ensure links are vaild and load the next page. 
     it('Generic menu items', () => {
-      cy.visitResponsive(Cypress.env('RegionEU'), 'small') //See custom command for details
+      cy.c_visitResponsive(Cypress.env('RegionEU'), 'small') //See custom command for details
```
Comment (habib-deriv) [inline cypress/e2e/smoke/home.cy.js:7]: Hi @markodonnell-deriv , can we remove this comment from all pages?
//See custom command for details

Maybe we can update the documentation, with this details for custom command

File: .DS_Store
Comment (markodonnell-deriv) [reply .DS_Store:1]: Yeah, it is in there. Not sure why it's not working

File: cypress/e2e/smoke/home.cy.js (hunk 1 above)
Comment (markodonnell-deriv) [reply cypress/e2e/smoke/home.cy.js:7]: Of course, I'll be doing another PR for perf tuning next and will revisit the comments then.

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**